src/main/java/org/apache/sysds/runtime/controlprogram/paramserv/LocalPSWorker.java

```diff
@@ -62,7 +64,13 @@ public Void call() throws Exception {
 
 			switch (_freq) {
 				case BATCH:
-					computeBatch(dataSize, batchIter);
+					if (_modelAvg){
+						computeBatch_Avg(dataSize,batchIter);
+					}
+
+					else
+						computeBatch(dataSize, batchIter);
```

Comment:
the formatting seems off again.

Reply:
Thanks for the comment. formatting fixed.